### DISCUSSION TITLE
why the vun-list page got http 500 anytime?

### DISCUSSION THREAD

**Discussion by zhuxindong** [aquasecurity/trivy#4500] [Q&A] [answered]
### Description

the page [https://github.com/aquasecurity/vuln-list/](https://github.com/aquasecurity/vuln-list/%E2%80%9D) got status 500 in anytime
![image](https://github.com/aquasecurity/trivy/assets/24989246/899c10b6-ae84-4453-9e77-fb9a32ce0a4e)

### Desired Behavior

the page “[https://github.com/aquasecurity/vuln-list/”](https://github.com/aquasecurity/vuln-list/%E2%80%9D) got status 500 in anytime

### Actual Behavior

the page “[https://github.com/aquasecurity/vuln-list/”](https://github.com/aquasecurity/vuln-list/%E2%80%9D) got status 500 in anytime

### Reproduction Steps

```bash
1. the page “[https://github.com/aquasecurity/vuln-list/”](https://github.com/aquasecurity/vuln-list/%E2%80%9D) got status 500 in anytime
2.
3.
...
```

### Target

None

### Scanner

None

### Output Format

None

### Mode

None

### Debug Output

```bash
the page “[https://github.com/aquasecurity/vuln-list/”](https://github.com/aquasecurity/vuln-list/%E2%80%9D) got status 500 in anytime
```

### Operating System

windows

### Version

```bash
the page “[https://github.com/aquasecurity/vuln-list/”](https://github.com/aquasecurity/vuln-list/%E2%80%9D) got status 500 in anytime
```

### Checklist

- [ ] Run `trivy --reset`
- [ ] Read [the troubleshooting](https://aquasecurity.github.io/trivy/latest/docs/references/troubleshooting/)

**Comment by knqyf263** [comment, marked as answer]
Too many commits seem to break GitHub. We're trying to fix it.
